### ISSUE TITLE
Feature Request: See what permissions are granted for each grain

### ISSUE THREAD

**Issue by ocdtrekkie** [dwrensha/sandstorm-collections-app#3]
In my case, I currently have a collection with three grains:

Etherpad, write access
Etherpad, read access
EtherCalc, read access

But there's no way for me to tell, for instance, that someone shared this collection can write to one file, but only read to the other two. I "just know" that, but if I forget, I could do something silly or wrong.

**Comment by dwrensha** [comment]
I agree that it would be nice to see what permissions are granted. Currently Sandstorm provides no way for a collection to access that information. Maybe we could add some methods to [`PersistentUiView`](https://github.com/sandstorm-io/sandstorm/blob/v0.174/src/sandstorm/persistentuiview.capnp#L24) to make the information available. New methods there might include `getCurrentPermissions()` and maybe even `getUpstreamGrainTitle()`.

**Comment by gwillen** [comment]
I was noticing the relevance of this to the Go game app, where this would be required to know whether the grain capability is to play as Black or as White. In general it seems like -- for capabilities more interesting than 'read' and 'write' -- it might be really limiting not to see the name of the capability in the list.
